RNTester/js/SwipeableListViewExample.js

```diff
@@ -100,45 +94,45 @@ var SwipeableListViewSimpleExample = createReactClass({
         </View>
       </TouchableHighlight>
     );
-  },
+  };
 
-  _genDataSource: function(pressData: {[key: number]: boolean}): Array<any> {
-    var dataBlob = {};
-    var sectionIDs = ['Section 0'];
-    var rowIDs = [[]];
+  _genDataSource(pressData: {[key: number]: boolean}): Array<any> {
+    const dataBlob = {};
+    const sectionIDs = ['Section 0'];
+    const rowIDs = [[]];
     /**
      * dataBlob example below:
-      {
-        'Section 0': {
-          'Row 0': {
-            id: '0',
-            text: 'row 0 text'
-          },
-          'Row 1': {
-            id: '1',
-            text: 'row 1 text'
-          }
-        }
-      }
-    */
+     *   {
+     *     'Section 0': {
+     *       'Row 0': {
+     *         id: '0',
+     *         text: 'row 0 text'
+     *       },
+     *       'Row 1': {
+     *         id: '1',
+     *         text: 'row 1 text'
+     *       }
+     *     }
+     *   }
+     */
     // only one section in this example
     dataBlob['Section 0'] = {};
-    for (var ii = 0; ii < 100; ii++) {
-      var pressedText = pressData[ii] ? ' (pressed)' : '';
+    for (let ii = 0; ii < 100; ii++) {
+      const pressedText = pressData[ii] ? ' (pressed)' : '';
       dataBlob[sectionIDs[0]]['Row ' + ii] = {
         id: 'Row ' + ii,
         text: 'Row ' + ii + pressedText,
       };
       rowIDs[0].push('Row ' + ii);
     }
     return [dataBlob, sectionIDs, rowIDs];
-  },
+  }
 
-  _renderSeperator: function(
-    sectionID: number,
-    rowID: number,
+  _renderSeperator = (
+    sectionID: string,
+    rowID: string,
     adjacentRowHighlighted: boolean,
-  ) {
+  ) => {
```

Comment:
Since we're not using `this` inside the `_renderSeparator` body, it's probably fine for us to use a regular function here. 

**Also:** This is not necessary, but since we're not using any instance state of `SwipeableListViewSimpleExample` objects, we don't even have to keep this an instance method.

Reply:
Thinking a top level function in the module, or a static method?